roles/openshift_storage_glusterfs/tasks/check_cluster_health.yml

```diff
@@ -0,0 +1,36 @@
+---
+# glusterfs_check_containerized is a custom module defined at
+# lib_utils/library/glusterfs_check_containerized.py
+- name: Check for cluster health of glusterfs
+  glusterfs_check_containerized:
```

Comment:
`run_once: true` on these to protect against someone running this against all hosts or something via new play down the road?

Reply:
Actually, we don't want run_once here; this task is executed (primarily) during the serial portion of the node upgrade.  We want to ensure that this runs for each node, this allows us to know the gluster volumes are healed before we move on to the next node.